docs/notebook-demos/catalog.ipynb

```diff
@@ -0,0 +1,614 @@
+    "\n",
+    "**Note**: It is not possible to save a layer to a catalog if the layer name and zoom already exist. If you wish to overwrite an existing, saved layer then it must be deleted before writing the new one.\n",
+    "\n",
+    "**Note**: Saving a `TiledRasterLayer` that does not have a `zoom_level` will save the layer to a zoom of 0. Thus, when it is read back out from the catalog, the resulting `TiledRasterLayer` will have a `zoom_level` of 0."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving a Spatial Layer\n",
+    "\n",
+    "Saving a spatial layer is a straight forward task. All that needs to be supplied is a `URI`, the name of the layer, and the layer to be saved."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# The zoom level which will be saved\n",
+    "spatial_tiled_layer.zoom_level"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This will create a catalog called, \"spatial-catalog\" in the /tmp directory.\n",
+    "# Within it, a layer named, \"spatial-layer\" will be saved.\n",
+    "gps.write(uri='file:///tmp/spatial-catalog', layer_name='spatial-layer', tiled_raster_layer=spatial_tiled_layer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving a Spatial Temporal Layer\n",
+    "\n",
+    "When saving a spatial-temporal layer, one needs to consider how the records within the catalog will be spaced; which in turn, determines the resolution of index. The `TimeUnit` enum class contains all available units of time that can be used to space apart data in the catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# The zoom level which will be saved\n",
+    "space_time_tiled_layer.zoom_level"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This will create a catalog called, \"spacetime-catalog\" in the /tmp directory.\n",
+    "# Within it, a layer named, \"spacetime-layer\" will be saved and each indice will be spaced apart by SECONDS\n",
+    "gps.write(uri='file:///tmp/spacetime-catalog',\n",
+    "          layer_name='spacetime-layer',\n",
+    "          tiled_raster_layer=space_time_tiled_layer,\n",
+    "          time_unit=gps.TimeUnit.SECONDS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving a Pyramid\n",
+    "\n",
+    "For those that are unfamiliar with the `Pyramid` class, please see the [Pyramid section] of the visualization guide. Otherwise, please continue on.\n",
```

Comment:
will this link work? I know you said you were changing how the docs are structured now